qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -43,10 +46,20 @@
     properties_from_server_data,
 )
 from qiskit_ibm_provider.utils import local_to_utc
+from qiskit_ibm_provider.utils import validate_job_tags, are_circuits_dynamic
+from qiskit_ibm_provider.utils.options import QASM2Options, QASM3Options
+from qiskit_ibm_provider.exceptions import IBMBackendValueError, IBMBackendApiError
+from qiskit_ibm_provider.api.exceptions import RequestsApiError
+
+# temporary until we unite the 2 Session classes
+from qiskit_ibm_provider.session import (
+    Session as ProviderSession,
+)  # temporary until we unite the 2 Session classes
```

Comment:
Yeah I think this is fine for now since we are not removing any functionality 

```python
backend = service.backend('ibmq_qasm_simulator')

with Session(service, backend=backend) as session:
    sampler = Sampler(session=session)
    
    job1 = sampler.run(bell)
    job2 = backend.run(bell) # this would not run in a session 

```
We can still use `backend.run()` in sessions like before but we should unite/consildate the classes so this code snippet works.

Reply:
If you agree, I'd rather leave this for a separate PR.